uninstall.sh

```diff
@@ -246,6 +246,7 @@ then
   [[ -n "${prefix}" ]] && homebrew_prefix_candidates+=("$(dirname "$(dirname "$(strip_s "${prefix}")")")")
   homebrew_prefix_candidates+=("${homebrew_prefix_default}") # Homebrew default path
   homebrew_prefix_candidates+=("${HOME}/.linuxbrew")         # Linuxbrew default path
+  [[ "$(uname -m)" == "arm64" ]] && homebrew_prefix_candidates+=("/usr/local") # If migrated from Intel to ARM old path will remain
```

Comment:
https://github.com/Homebrew/install/runs/8200050551?check_suite_focus=true#step:16:56

Reply:
I seen but I have no clue what your style settings are :( .. 
it's a one-liner, no clue what style rule is broken

Reply:
Run `brew style --fix .` on this directory when checked out.

Reply:
> Run `brew style --fix .` on this directory when checked out.

tried it when I seen the error report from the CI but nothing happens:

```
minimac-m1:install bk$ brew style --fix
Inspecting 1106 files
..................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................

1106 files inspected, no offenses detected
minimac-m1:install bk$ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
minimac-m1:install bk$ git diff
minimac-m1:install bk$
```


**1106 files inspected, no offenses detected**

Reply:
hm looks like if no input files are given fix check who knows what :D ... should be ok now :) I see it moved comments in 2 lines above